sqlmesh/core/context.py

```diff
@@ -728,6 +678,7 @@ def plan(
         no_auto_categorization: t.Optional[bool] = None,
         effective_from: t.Optional[TimeLike] = None,
         include_unmodified: t.Optional[bool] = None,
+        model_selections: t.Optional[t.Iterable[str]] = None,
```

Comment:
i think you want a collection instead, because you can't check if an iterable is empty or not